app/assets/javascripts/components/input_table.ts

```
@@ -0,0 +1,333 @@
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify({
+                evaluation_exercise: {
+                    visible_score: this._totalVisible,
+                    score_items: this.editedScoreItems
+                }
+            })
+        });
+        if (response.ok) {
+            const js = await response.text();
+            eval(js);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)